waterloop-site/src/components/PastGeeseTimeline/hooks/geese-images.ts

```diff
@@ -64,13 +66,19 @@ const useGeeseImages: GooseImagesHook = () => {
     [setCurrentIndex, currentIndex]
   );
 
+        
+  const selectGoose = (index: number) => 
+    setCurrentIndex(index)
+    
```

Comment:
```
const selectGoose = useCallback((index: number) => () => setCurrentIndex(index), [setCurrentIndex]);
```

Reply:
Also changed, but it still does not match the type for callback 
![image](https://user-images.githubusercontent.com/71536798/123526585-e16f1200-d6a6-11eb-81fe-e7d46707572c.png)

Reply:
![image](https://user-images.githubusercontent.com/71536798/123526620-25621700-d6a7-11eb-8ddc-08cae7604f14.png)

Used hook as suggested.